### DISCUSSION TITLE
Contact (I'm a grey hat hacker)

### DISCUSSION THREAD

**Discussion by vezcodes** [Discord-Oxygen/Discord-Oxygen#17] [General]
Hello dear HXR404,

I've researched your repo completely and, wow, I'm impressed. I do grey hat hacking, also known as ethical hacking and would love to get in touch with you. Please consider contacting me trough Discord!
Sfyx#1222

Hopefully I'll meet you there :)

**Comment by steinhorror** [comment]
we already talked

**Reply by Arsha915** [reply]
Can u help me
